Quaver.Shared/Screens/ConfirmDialog.cs

```diff
@@ -0,0 +1,236 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ * Copyright (c) 2017-2018 Swan & The Quaver Team <support@quavergame.com>.
+*/
+
+using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+using Quaver.Shared.Assets;
+using Quaver.Shared.Graphics;
+using Wobble.Graphics;
+using Wobble.Graphics.Primitives;
+using Wobble.Graphics.Sprites;
+using Wobble.Graphics.UI.Buttons;
+using Wobble.Graphics.UI.Dialogs;
+using Wobble.Input;
+using Wobble.Logging;
+using Wobble.Window;
+
+namespace Quaver.Shared.Screens
+{
+    public class ConfirmDialog : DialogScreen
+    {
+        /// <summary>
+        ///     The box where all the content is contained.
+        /// </summary>
+        public Sprite ContainingBox { get; set; }
+
+        /// <summary>
+        ///     The top line of the containing box.
+        /// </summary>
+        public Line TopLine { get; set; }
+
+        /// <summary>
+        ///     Text that asks the user if they're sure they want to quit.
+        /// </summary>
+        public SpriteText AreYouSure { get; set; }
+
+        /// <summary>
+        ///     The line that divides AreYouSure and the buttons.
+        /// </summary>
+        public Line DividerLine { get; set; }
+
+        /// <summary>
+        ///     The button to confirm the action.
+        /// </summary>
+        public TextButton ConfirmButton { get; set; }
+
+        /// <summary>
+        ///     The button to cancel the dialog.
+        /// </summary>
+        public TextButton CancelButton { get; set; }
+
+        /// <summary>
+        ///     Event to verify confirmation.
+        /// </summary>
+        public event EventHandler<bool> Confirmation;
+
+        private string AreYouSureText { get; }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// </summary>
+        public ConfirmDialog(string areYouSureText) : base(0.75f)
+        {
```

Comment:
We should keep dialog animations consistent. Check out the [SettingsDialog](https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Screens/Settings/SettingsDialog.cs#L112).

It adds a fade animation rather than starting at 0.75 alpha.